ArchiSteamFarm/Bot.cs

```diff
@@ -1073,6 +1073,16 @@ internal sealed class Bot : IDisposable {
 								return await ResponseAdvancedLoot(steamID, args[1], args[2]).ConfigureAwait(false);
 							}
 
+							goto default;
+						case "LOOT@":
+							if (args.Length > 2) {
+								return await ResponseLootByRealAppID(steamID, args[1], Utilities.GetArgsAsText(args, 2, ",")).ConfigureAwait(false);
+							}
+
+							if (args.Length == 2) {
```

Comment:
This `if` is redundant since the switch above guarantees that you'll have at least 2 args.